packages/twenty-server/src/modules/calendar/calendar-event-import-manager/drivers/utils/sanitizeCalendarEvent.ts

```diff
@@ -0,0 +1,24 @@
+import { isDefined } from 'twenty-shared/utils';
+
+export const sanitizeCalendarEvent = <T extends Record<string, any>>(
+  event: T,
+  propertiesToSanitize: (keyof T)[],
+): T => {
+  const sanitizedEvent = { ...event };
+
+  for (const property of propertiesToSanitize) {
+    if (!isDefined(sanitizedEvent[property])) {
+      continue;
+    }
+    if (typeof sanitizedEvent[property] !== 'string') {
+      continue;
+    }
+    sanitizedEvent[property] = sanitizeString(sanitizedEvent[property]) as any;
```

Comment:
why do we need this as any here?

Reply:
We need something (otherwise TypeScript complains) but i norrowsed down the types to make it better in my last commit